spk/sonarr3/src/config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
Just a small suggestion. There is no <?xml> tag after sonarr has updated this file. So we can firmly remove this line.
```suggestion
```